api-samples/declarativeNetRequest/no-cookies-manager/service_worker.js

```diff
@@ -0,0 +1,45 @@
+chrome.runtime.onInstalled.addListener(async function () {
+  // restore the default rule if the extension is installed or updated
+  const _rules = await chrome.declarativeNetRequest.getDynamicRules();
```

Comment:
I think we can just call this rules or existingRules rather than needing the underscore?

Reply:
Done. https://github.com/GoogleChrome/chrome-extensions-samples/pull/986/commits/4b52b3dd127657998006b6318925836ba958f248